### PR TITLE
Add release note for 1.2.39 version

### PR DESCRIPTION
**Pull request by SasinduDilshara**
${subject}

### CODE REVIEW

File: _data/release_notes_versions.json [hunk 1]
```diff
@@ -1200,5 +1200,23 @@
...
+      "windows-installer-size":"148mb",
+      "linux-installer":"ballerina-linux-installer-x64-1.2.39.deb",
+      "linux-installer-size":"145mb",
+      "macos-installer":"ballerina-macos-installer-x64-1.2.39.pkg",
+      "macos-installer-size":"164mb",
+      "rpm-installer":"ballerina-linux-installer-x64-1.2.39.rpm",
+      "rpm-installer-size":"167mb",
+      "other-artefacts":[  
+         "ballerina-1.2.39.zip",
+         "ballerina-1.2.39.vsix"
```
Comment (chiranSachintha) [inline _data/release_notes_versions.json:1217]: Are we planning to release VSIX version 1.2.39?

Comment (chiranSachintha) [reply _data/release_notes_versions.json:1217]: I think we need to remove that. We have not released any versions after 1.2.20.

**Review by sm1990 — APPROVED** [review]

File: _data/release_notes_versions.json (hunk 1 above)
Comment (SasinduDilshara) [reply _data/release_notes_versions.json:1217]: This json is generated one, So I think it is ok to have that, However frontend will remove this.
This is the same approach odf 1.2.38

Comment (NipunaMadhushan) [reply _data/release_notes_versions.json]: @SasinduDilshara remove it.

**Review by praneesha — APPROVED** [review]

**Pull request merged**